demand_acep/demand_acep.py

```diff
@@ -13,16 +13,29 @@
 
 # %%
 
-def extract_data(dirpath, filename, channel):
+def hello_world():
+    """
+    This function returns the string Hello World.
+
+    This is for testing the CI and other devops functionality.
+
+    :return: A string "Hello world!"
+    """
+
+    print("Hello WOrlds")
+
+
+    return "Hello world!"
+
+
+def extract_data(dirpath, filename):
     """This function takes in a filename and directory path, extracts the meter channel data in that file and
     returns the time and channel values at each time"""
     netcdf_data = xr.open_dataset(os.path.join(dirpath, filename))
     netcdf_df = netcdf_data.to_dataframe()
-    netcdf_df.columns = [channel]
     netcdf_df.set_index(pd.to_datetime(netcdf_df.index.values), inplace=True)
-    netcdf_resampled = netcdf_df.resample('1T').mean()
     # pdb.set_trace()
-    return netcdf_resampled
+    return netcdf_df
```

Comment:
So, instead of a list, this is a dataframe now. Only the data column does not have a name yet.